src/ol_infrastructure/applications/mitopen/Pulumi.applications.mitlearn.QA.yaml

```diff
@@ -16,19 +16,19 @@ config:
     INDEXING_API_USERNAME: "od_mm_rc_api"
     KEYCLOAK_BASE_URL: "https://sso-qa.ol.mit.edu"
     KEYCLOAK_REALM_NAME: "olapps"
-    MITOPEN_API_URL: "https://api.mitopen-rc.odl.mit.edu"
-    MITOPEN_APP_URL: "https://mitopen-rc.odl.mit.edu"
-    MITOPEN_BASE_URL: "https://mitopen-rc.odl.mit.edu"
-    MITOPEN_COOKIE_DOMAIN: "odl.mit.edu"
-    MITOPEN_COOKIE_NAME: "mitopen-rc"
+    MITOPEN_API_URL: "https://api.rc.learn.mit.edu"
+    MITOPEN_APP_URL: "https://rc.learn.mit.edu"
+    MITOPEN_BASE_URL: "https://rc.learn.mit.edu"
+    MITOPEN_COOKIE_DOMAIN: "learn.mit.edu"
```

Comment:
Should we make this `rc.learn.mit.edu`?

Reply:
Probably, to avoid mingling production and RC cookies.